### PR TITLE
Add shortcut for Copy in Map editor

### LINKED ISSUE #14723: Map Editor : Add shortcuts
Especially a Ctrl-C shortcut for Copy/paste
Maybe G for Grid. 

#14724 : Ctrl-D for Clone tool

### PR DESCRIPTION
**Pull request by rob-v**
Add Ctrl+C shortcut for Copy in Map editor
(Grid already has F1 key)

Closes #14723

### CODE REVIEW

**Comment by pchote** [comment]
I found two issues while testing this:
* The Copy/Paste button is missing the tooltip which shows the shortcut
* The shortcut needs to be Cmd + C to match platform conventions on macOS.

**Comment by rob-v** [comment]
'Cmd' is never used in a .yaml and I couldn't get it working on Windows. We use also elsewhere 'Ctrl'. Can I keep Ctrl or what should I do?

**Comment by rob-v** [comment]
Updated - added tooltip for Copy and Zoom.

**Comment by pchote** [comment]
> 'Cmd' is never used in a .yaml and I couldn't get it working on Windows. We use also elsewhere 'Ctrl'. > Can I keep Ctrl or what should I do?

The modifier is called `Meta` internally - find references to see the other cases where we special-case <kbd>Ctrl</kbd> &rarr; <kbd>Cmd</kbd>.

See 719c747cda64f9a849ec8d61ecb37e40793ea685 for IMO the best short-term approach.
Fixing this properly is going to be a much bigger job.

**Review by pchote — APPROVED** [review]
LGTM, but best to wait for another @OpenRA/engine-hackers to approve my hotkey hack before merging.

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/f2beec7000aada4ce444a4258d6e303d92b4ca61

de851fba2c0cb124c704ca22def78e0cf2262031
9025d11c54ad4a0aec537d917b22a6f8a68914c9

**Pull request merged**
